util/src/main/i18n/templates/moderation.properties

```diff
@@ -150,6 +150,10 @@ moderation.defuse.enemy = You may not defuse enemy TNT.
 
 moderation.defuse.displayName = TNT Defuser
 
+# {0} = defuser player name
+# {1} = griefer player name
+moderation.defuse.message = {0} defused {1}'s TNT!
```

Comment:
I would remove the exclamation mark. Don't think any other messages have that

Reply:
Look down like 4 translations in the file. It's pretty uncommon to not use them, tough.

Reply:
Alright, fair enough. Still believe that we should remove all exclamation points cause it looks unprofessional, but that's a discussion for another time.

Reply:
I like it with the exclamation mark ❗

Reply:
Translations should avoid punctuation, so let's not use the "!"

Reply:
Why would translations explicitly not use punctuation?

Reply:
I removed the exclamation mark from the message